package.json

```diff
@@ -32,14 +32,16 @@
     ]
   },
   "devDependencies": {
+    "@types/node": "^20.8.0",
```

Comment:
FWIW, this should be `^18.x.y` (for the latest x and y). Otherwise, the TypeScript compiler will allow us to use APIs introduced in Node.js 20 that are not available in Node.js 18 yet.